crates/turborepo-lib/src/cli/error.rs

```diff
@@ -53,3 +55,51 @@ pub enum Error {
     #[diagnostic(transparent)]
     Watch(#[from] watch::Error),
 }
+
+const MAX_CHARS_PER_TASK_LINE: usize = 100;
+
+pub async fn print_potential_tasks(
+    base: CommandBase,
+    telemetry: CommandEventBuilder,
+) -> Result<(), Error> {
+    let signal = get_signal()?;
+    let handler = SignalHandler::new(signal);
+    let ui = base.ui;
+
+    let run_builder = RunBuilder::new(base)?;
+    let run = run_builder.build(&handler, telemetry).await?;
+    let potential_tasks = run.get_potential_tasks()?;
+
+    println!("No tasks provided, here are some potential ones to run\n",);
+
+    for (task, packages) in potential_tasks
+        .into_iter()
+        .sorted_by(|(_, a), (_, b)| b.len().cmp(&a.len()))
+    {
+        let task = color!(ui, BOLD, "{}", task);
+        let mut line_length = 0;
+
+        let mut packages_str = String::with_capacity(MAX_CHARS_PER_TASK_LINE);
+        for (idx, package) in packages.iter().sorted().enumerate() {
+            if line_length > MAX_CHARS_PER_TASK_LINE {
```

Comment:
note: maybe it's totally arbitrary and fine, but I thought I'd mention that I noticed that if you run this (in this repo, for example) you get a 32 characters over the `MAX_CHARS_PER_TASK_LINE` limit:

```
    @turbo/benchmark, @turbo/codemod, @turbo/telemetry, @turbo/test-utils, @turbo/types, @turbo/utils, @turbo/workspaces and 10 more
```

From reading the code this seems to be because:
- the `    ` indent (four initial spaces) isn't counted to the max
- the ` and 10 more` isn't counted (which is effectively unbounded in length, but practically probably never more than 3 chars)
- so in this case the first few packages (`@turbo/benchmark, @turbo/codemod, @turbo/telemetry, @turbo/test-utils, @turbo/types, @turbo/utils`) are 97 chars in length and since that's under the limit it adds the next one, which adds another 18 chars (`, @turbo/workspaces`).

Reply:
Yeah I was kinda loose on this, we can make it a hard 80 char restriction but tbh it was more like "keep it somewhat reasonable" since I doubt people are actually using exactly 80 char width terminals

Reply:
sweet.  thanks for clarifying :)